e2e/worker/test.sh

```diff
@@ -0,0 +1,68 @@
+set -o errexit
+
+cd $( dirname -- "$BASH_SOURCE"; )
+
+
+message() {
+    echo "###########################"
+    echo "$1"
+}
+
+exit_with_message() {
+    message "$1"
+    exit 1
+}
+
+cleanup() {
+    rm -f evil.ts included.ts
+    git checkout HEAD -- tsconfig.json
+}
+
+# just run it in case there is leftovers
+cleanup
+# set up trap for graceful cleanup
+trap cleanup EXIT
+
+
+message "# Case 0; try to build a target that will never succeed"
+message="error TS2322: Type 'number' is not assignable to type 'string'."
+bazel build :should_fail 2>&1 | grep "$message" || exit_with_message "Case 0: expected worker to report \"$message\" but it didn't"
```

Comment:
I think we can drop the "but it didn't"? Maybe put "FAIL: " at the start?

Reply:
Done. just changed `exit_with_message` func